src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -366,8 +370,8 @@ class FunSpec private constructor(builder: Builder) {
     internal const val GETTER = "get()"
     internal const val SETTER = "set()"
 
-    internal val String.isConstructor get() = this == CONSTRUCTOR
-    internal val String.isAccessor get() = this.isOneOf(GETTER, SETTER)
+    internal val String?.isConstructor get() = this == CONSTRUCTOR
```

Comment:
`===` ?

Reply:
Good call!

Reply:
..although technically this is what `String.equals()` already does, but semantically I like `===` better